free-programming-books.md

```diff
@@ -634,6 +659,7 @@
 * [HTTP Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/http) (PDF, Kindle) (email address *requested*, not required)
 * [HTTP2 Explained](http://daniel.haxx.se/http2/) - Daniel Stenberg
 * [IPv6 for IPv4 Experts](https://sites.google.com/site/yartikhiy/home/ipv6book) - Yar Tikhiy (PDF)
+* [Network Science](http://networksciencebook.com/) - Albert-Laszló Barabási
```

Comment:
trailing slash. interesting book

Reply:
also, translated versions could be added to other lists - maybe create an issue for native speakers to add.

Reply:
I actually went ahead and added them!